### PR TITLE
Update this plugin to use the official hipchat gem

### LINKED ISSUE #7: No tests / Tests missing
Hi,
  this plugin has no test at all, this is a situation that should be amended soon, so for example changes in both the plugin or the logstash core himself, are properly tested in the CI environment.

If you are giving this plugin some test, feel free to update https://github.com/elastic/logstash/issues/3740 so we keep track of it.

### PR DESCRIPTION
**Pull request by ph**
- Do not use ftw anymore
- Dropped dependency on the http output
- Allow people to use the v2 api and hipchat's cloud offering

Fixes #7  #5 

Also includes suggestion from #4 #3

### CODE REVIEW

**Comment by ph** [comment]
I did not merge #4, #3 since the CLA isn't sign and the code diverged a lot.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 1.0.1
```
Comment (suyograo) [inline CHANGELOG.md]: Can you bump this to 2.0.0? not sure the old v1 APIs are backward compatible

File: logstash-output-hipchat.gemspec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-hipchat'
-  s.version         = '1.0.0'
+  s.version         = '1.0.1'
```
Comment (suyograo) [inline logstash-output-hipchat.gemspec]: Can you bump this to 2.0.0? not sure the old v1 APIs are backward compatible

File: CHANGELOG.md (hunk 1 above)
Comment (ph) [inline CHANGELOG.md]: good catch!

File: lib/logstash/outputs/hipchat.rb [hunk 3]
```diff
@@ -22,61 +23,49 @@ class LogStash::Outputs::HipChat < LogStash::Outputs::Base
...
     return unless output?(event)
 
-    hipchat_data = Hash.new
-    hipchat_data['room_id'] = event.sprintf(@room_id)
-    hipchat_data['from']    = @from
-    hipchat_data['color']   = @color
-    hipchat_data['notify']  = @trigger_notify ? "1" : "0"
-    hipchat_data['message'] = event.sprintf(@format)
+    message = event.sprintf(@format)
+    from = event.sprintf(@from)
```
Comment (suyograo) [inline lib/logstash/outputs/hipchat.rb]: nice.. good call on using `sprintf`for all fields

**Comment by ph** [comment]
@suyograo updated with your changes!

**Comment by suyograo** [comment]
LGTM

**Pull request closed without merging**